libpod/image/image.go

```diff
@@ -1151,3 +1151,30 @@ func (i *Image) Save(ctx context.Context, source, format, output string, moreTag
 
 	return nil
 }
+
+// GetConfigBlob returns a schema2image
```

Comment:
Should we note that this only works for Schema2 images, and fails for other types?

Reply:
we are going to still implement healthcheck for other types, it just cannot originate from the image itself.  new run options!

Reply:
But this is in libpod/image. We can't source healthchecks from anything except schema2.